tools/templates/DeviceBindingTemplate/dotnet_new_device-binding_csharp/tests/_DeviceBinding_Not_Required.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Can you add the note saying this is also not required and can be removed here?

Reply:
Added note.